### DISCUSSION TITLE
Google cloud: "7 days left" - trial expiring

### DISCUSSION THREAD

**Discussion by emp-00** [NightscoutFoundation/xDrip#2547] [Q&A] [answered]
I have set up my nightscout as per @Navid200 's instructions on google cloud a few months ago, all running fine so far.

Now, I just received an urgent email from GC saying _"7 days left: There  are still seven days until your trial expires. Upgrade your account now  to enable automatic billing and prevent loss of service when your trial  expires."_

Clicking on the link I see this: _"Activate your full account, Keep your cloud running uninterrupted, Keep any remaining credits to spend during your Free Trial, Pay only for what you use—automatic billing starts once your Free Trial ends"_

What now? Do I really have to activate my account or otherwise my nightscout in the cloud will be switched off? I just would like it to continue with no costs - I'm aware that there was a "trial period" but I thought I can just leave it running after the trial ends. Has anybody already passed this "milestone" and give me some advice?

**Comment by Navid200** [comment, marked as answer]
Yes, you should activate as explained here:
https://navid200.github.io/xDrip/docs/Nightscout/FreeTrialEnd.html

**Reply by emp-00** [reply]
Chapeau - Thanks!!
